### PR TITLE
Single Post Image AM-111

### PR DESCRIPTION
**Pull request by Dea523**

### CODE REVIEW

File: lib/singlePostImage.dart [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    return SafeArea(
+      child: Scaffold(
+          body: Column(
+        // mainAxisAlignment: MainAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Icon(Icons.arrow_back_rounded),
```
Comment (abhishek2355) [inline lib/singlePostImage.dart:28]: Add size parameter in icon -->  Icon(Icons.arrow_back_rounded, size: screenHeight * app_heights.height25), like this one, do the same in the complete file.

Comment (abhishek2355) [reply lib/singlePostImage.dart:28]: Do the same for the text widget also.

File: lib/constants/app_strings.dart [hunk 2]
```diff
@@ -189,6 +189,21 @@ const yesterday = "Yesterday";
...
 const last7Days = "Last 7 Days";
 
+//singlePost Image
+//Username bring it from notification page
+const like = "Like";
+const comment = "Comment";
+const repost = "Repost";
+const send = "Send";
+const reaction = "Reaction";
+// const comments = "Comments";
```
Comment (omkarsantoshraut) [inline lib/constants/app_strings.dart:199]: Remove unwanted changes/comments.

File: lib/singlePostImage.dart [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+    final double screenHeight = deviceMedia.size.height -
+        deviceMedia.padding.top -
+        deviceMedia.padding.bottom;
+    final double screenWidth = deviceMedia.size.width -
+        deviceMedia.padding.left -
+        deviceMedia.padding.right;
+    return SafeArea(
+      child: Scaffold(
+          body: Column(
+        // mainAxisAlignment: MainAxisAlignment.start,
```
Comment (omkarsantoshraut) [inline lib/singlePostImage.dart:22]: Remove this if it is unwanted.

File: lib/singlePostImage.dart [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+            height: screenHeight * app_heights.height20,
+          ),
+          Container(
+            width: screenWidth * app_widths.width425,
+            height: screenHeight * app_heights.height30,
+            color: Colors.grey.shade200,
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              children: [
+                Icon(Icons.heart_broken),
```
Comment (omkarsantoshraut) [inline lib/singlePostImage.dart:51]: I think we already have our widget for this - @abhishek2355 can replace with that widget before creating the final PR.

**Review by omkarsantoshraut — CHANGES_REQUESTED** [review]

File: lib/singlePostImage.dart (hunk 1 above)
Comment (Dea523) [reply lib/singlePostImage.dart:28]: om i will do that

File: lib/constants/app_strings.dart (hunk 2 above)
Comment (Dea523) [reply lib/constants/app_strings.dart:199]: yes

**Review by abhishek2355 — APPROVED** [review]
Good

**Pull request merged**